### DISCUSSION TITLE
Support for mTLS / certificate authentication

### DISCUSSION THREAD

**Discussion by Finkregh** [LeoKlaus/plappa#205] [Feature Requests]
It would be nice if I could authenticate the client with a certificate for additional peace of mind.

Example solution from another application: https://github.com/immich-app/immich/pull/10860

**Comment by andre-sam** [comment]
Currently, Immich allows configuring client certificates (p12 and pfx) before login, while Plappa supports adding custom headers. It would be awesome to have an option in Plappa to "import client certificate" for mTLS, similar to Immich.
